### PR TITLE
About page gif

### LINKED ISSUE #477: Change odin logo in contact section of about page to provided gif
This task is to replace the Odin-logo image in the contact section of the [about page](https://staging-odin-project.herokuapp.com/about) with the `odin-wink.gif` which can be found under the assets for the about page.

This is the section in question:
screen shot 2017-04-30 at 14 48 02

### Working on this task

- [ ] If you want to work on this task, please claim it by commenting below   
- [ ] Branch off the redesign branch                                                                 
- [ ] When you are submitting your pull request for this task please ensure the base branch you want to merge into is the redesign branch

### PR DESCRIPTION
**Pull request by codyloyd**
fixes #477

### CODE REVIEW

**Comment by codyloyd** [comment]
lol I have to change the base every time 🙄

**Comment by KevinMulhern** [comment]
👍 LGTM

File: app/views/static_pages/home/_success_stories.html.erb [hunk 1]
```diff
@@ -13,8 +13,8 @@
   %>
 
   <div class="success-stories-container">
-    <% success_stories.each do |story| %>
-      <div class="success-story">
+    <% success_stories.each_with_index do |story, index| %>
+      <div class="success-story <%= "story-#{index}"%>">
```
Comment (arun1595) [inline app/views/static_pages/home/_success_stories.html.erb]: Cool hack ;)

**Comment by codyloyd** [comment]
I think there is plenty of padding between them now.

**Comment by adachiu** [comment]
One more little thing, can you balance the line length of "Connect with our
friendly..." text. Second line could start with "a chat ..."

Either that, or we update our copy and add a few more words to the second
line.

(Same goes to the bottom "Help the Odin Poject stay..." 🙈First line is too
long and text needs more line height. This could be another issue)

**Comment by codyloyd** [comment]
done!

**Pull request merged**